### DISCUSSION TITLE
월드 개발 코드 공유

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#315] [5. Q&A]
얼마전에 1월에 코드를 공개하신다는 글을 여기서 봤는데.. 공개되었나요?

공개되었다면 어디서 볼 수 있을 까요?

**Comment by x4cf** [comment]
_**=저는 제페토 내부 인원이 아닙니다 일반 유저입니다=**_
하하 공식에서는 정확한 일정이 나온적 없으니 아마 제가 올린글을 보셨던걸로 예상합니다.
튜토리얼과 관련 샘플프로젝트를 1월에 배포하려 했는데 조금 시간이 더 걸리는군요 :)
작업이 다 끝나가고 있어요 잠시만 기다려 주세요 :D

**Reply by leebeomchan** [reply]
1월에 공개해주시는 프로젝트가 제페토 공식 샘플 프로젝트인가요?

**Reply by x4cf** [reply]
아닙니다. 저는 일반 유저이고, 유저로써 샘플 프로젝트를 만들고 배포할 예정입니다.

**Reply by leebeomchan** [reply]
와우 좋은 일 하시네요 감사합니다.

**Comment by peterkangdev** [comment]
샘플에 서버 storage에 저장하는 방법도 올려주시면 좋겠네요..

**Reply by x4cf** [reply]
제가 만들었던 치즈버거서버 프로젝트를 그대로 샘플 프로젝트로 배포할 예정이에요 :)
저장이나 상점, 인벤토리등이 구현되어 있으니 한번 구경오세요 XD
